docker/supervisord.conf

```diff
@@ -0,0 +1,32 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/stdout
+logfile_maxbytes=0
+pidfile=/tmp/supervisord.pid
+
+[supervisorctl]
+serverurl=unix:///tmp/supervisorctl.sock
+
+[unix_http_server]
+file=/tmp/supervisorctl.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:uwsgi]
+command=/app/venv/bin/uwsgi --ini uwsgi_shared.ini --socket /srv/pajbot/%(ENV_STREAMERNAME)s-web.sock --log-x-forwarded-for --pyargv "--config /app/config.ini"
```

Comment:
`--log-x-forwarded-for` could probably also be put into `uwsgi.ini`, seems like a good option for normal nginx operation too